bugalink-frontend/next.config.js

```diff
@@ -12,4 +12,8 @@ module.exports = withPWA({
 
     return config;
   },
+  images: {
+    domains: ['localhost', '127.0.0.1', 'bugalink.es'],
```

Comment:
The domain is actually `app.bugalink.es`, this might not work in the deployment. Keep an eye on it